lib/rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] By adding attribute accessors for ```book``` and ```:person``` in the Rental class, you make it possible to interact with and manipulate these attributes from outside the class, which is essential for working with objects in a meaningful way. It provides a convenient interface for accessing and updating the data associated with a Rental object, This is a common practice in object-oriented programming.